server/api/sectionevent/sectionevent.controller.js

```diff
@@ -81,7 +81,12 @@ exports.index = function(req, res) {
       query.populate("author");
     }
     if (withEventInfo){
-      query.populate("info");
+      query.populate({
+        path:'info',
+        populate: {
+          path: 'isHappeningNow isPastDate'
```

Comment:
i didn't know you could add paths like this- that's awesome (and definitely means I should refactor a couple things)